doc/releases/release-notes-4.1.rst

```diff
@@ -24,359 +47,178 @@ https://docs.zephyrproject.org/latest/security/vulnerabilities.html
+
+* Other
+
+  * :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT_EXTRA`
+  * :c:macro:`DT_ANY_INST_HAS_BOOL_STATUS_OKAY`
+  * :c:struct:`led_dt_spec`
+
+New Boards & SoC
+****************
+..
+  Just link the board, details go in the board description
+
+* :zephyr:board:`max78002evkit`
+* :zephyr:board:`acp_6_0_adsp`
+* :zephyr:board:`ucan`
+* :zephyr:board:`ttgo_t7v1_5`
+* :zephyr:board:`ttgo_t8s3`
+* :zephyr:board:`m5stack_cores3`
+* :zephyr:board:`mks_canable_v20`
+* :zephyr:board:`octopus_io_board`
+* :zephyr:board:`octopus_som`
+* :zephyr:board:`candlelight`
+* :zephyr:board:`candlelightfd`
+* :zephyr:board:`xiao_esp32c6`
+* :zephyr:board:`rp2040_zero`
+* :zephyr:board:`ch32v003evt`
+* :zephyr:board:`we_orthosie1ev`
+* :zephyr:board:`mini_stm32h7b0`
+
+New Drivers
+***********
+..
+  Just link the driver, details go in the binding description
+
+* :dtcompatible:`silabs,dbus-pinctrl`
+
+* :dtcompatible:`renesas,bt-hci-da1453x`
+* :dtcompatible:`st,hci-stm32wb0`
+* :dtcompatible:`nuvoton,npcm-pcc`
+* :dtcompatible:`wch,ch32v00x-hse-clock`
+* :dtcompatible:`wch,ch32v00x-hsi-clock`
+* :dtcompatible:`wch,ch32v00x-pll-clock`
+* :dtcompatible:`wch,rcc`
+* :dtcompatible:`adi,max32-rtc-counter`
+* :dtcompatible:`wch,qingke-v2`
+* :dtcompatible:`xlnx,axi-dma-1.00.a`
+* :dtcompatible:`xlnx,eth-dma`
+* :dtcompatible:`renesas,ra-ethernet`
+* :dtcompatible:`lattice,ice40-fpga-base`
+* :dtcompatible:`lattice,ice40-fpga-bitbang`
+* :dtcompatible:`awinic,aw9523b-gpio`
+* :dtcompatible:`ite,it8801-gpio`
+* :dtcompatible:`nordic,npm2100-gpio`
+* :dtcompatible:`stemma-qt-connector`
+* :dtcompatible:`wch,gpio`
+* :dtcompatible:`nxp,hdlc-rcp-if`
+* :dtcompatible:`ti,tca9544a`
+* :dtcompatible:`ti,tca9544a-channel`
+* :dtcompatible:`ite,it8801-kbd`
+* :dtcompatible:`wch,pfic`
+* :dtcompatible:`linaro,ivshmem-mbox`
+* :dtcompatible:`renesas,ra-mdio`
+* :dtcompatible:`awinic,aw9523b`
+* :dtcompatible:`ite,it8801-altctrl`
+* :dtcompatible:`ite,it8801-mfd-map`
+* :dtcompatible:`ite,it8801-mfd`
+* :dtcompatible:`nordic,npm2100`
+* :dtcompatible:`nordic,nrf-ppib`
+* :dtcompatible:`fujitsu,mb85rsxx`
+* :dtcompatible:`silabs,dbus-pinctrl`
+* :dtcompatible:`wch,afio`
+* :dtcompatible:`ite,it8801-pwm`
+* :dtcompatible:`nordic,npm2100-regulator`
+* :dtcompatible:`adi,adxl366`
+* :dtcompatible:`adi,adxl366`
+* :dtcompatible:`hc-sr04`
+* :dtcompatible:`nordic,npm2100-vbat`
+* :dtcompatible:`sensirion,scd40`
+* :dtcompatible:`sensirion,scd41`
+* :dtcompatible:`sensirion,sts4x`
+* :dtcompatible:`wch,usart`
+* :dtcompatible:`ite,it8xxx2-spi`
+* :dtcompatible:`renesas,ra-spi`
+* :dtcompatible:`vnd,i2s`
+* :dtcompatible:`mediatek,ostimer64`
+* :dtcompatible:`wch,systick`
+* :dtcompatible:`ambiq,usb`
+* :dtcompatible:`zephyr,video-emul-imager`
+* :dtcompatible:`zephyr,video-emul-rx`
+* :dtcompatible:`nordic,npm2100-wdt`
+
+New Samples
+***********
 
-* Made these board changes:
+..
+  Just link the samples, details go in the sample documentation itself
```

Comment:
```suggestion
    Same as above for boards and drivers, this will also be recomputed at the time of the release.
    Just link the sample, details go in the sample documentation itself.
```

Reply:
done